packages/ChangesReloaded-Core.package/CRBrowser.class/instance/includeButtonProps.st

```diff
@@ -0,0 +1,4 @@
+building
+includeButtonProps
+	
+	^ #('Include' #toggleExcludeInclude 'Exclude/include selected change for getting loaded into the image')
```

Comment:
I don't really like that the tooltip is the same string duplicated in `excludeButtonProps` and `includeButtonProps` but only one of those strings is actually used. Maybe the tooltip should be normalized into a `includeExcludeButtonTooltip` or something like that?